test/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectVerifier.kt

```diff
@@ -82,7 +82,7 @@ class RedirectVerifier(responseDom: Node, parsedResponse: Map<String, String>, g
                 SigErrorCode.INVALID_CERTIFICATE -> throw SAMLComplianceException.create(SAMLBindings_3_1_2_1, message = "The certificate was invalid.", cause = e)
                 SigErrorCode.SIG_ALG_NOT_PROVIDED -> throw SAMLComplianceException.create(SAMLBindings_3_4_4_1_d1, message = "Signature Algorithm not found.", cause = e)
                 SigErrorCode.SIGNATURE_NOT_PROVIDED -> throw SAMLComplianceException.create(SAMLBindings_3_4_4_1_f2, message = "Signature not found.", cause = e)
-                SigErrorCode.INVALID_URI -> throw SAMLComplianceException.create(SAMLBindings_3_4_4_1_d2, message = "The Signature algorithm named $sigAlg is unknown.", cause = e)
+                SigErrorCode.INVALID_URI -> throw SAMLComplianceException.create(SAMLBindings_3_4_4_1_d2, message = "The Signature algorithm named $response.sigAlg is unknown.", cause = e)
```

Comment:
❗️ Could you manually test this message by forcing it? If IntelliJ is any indication, this will print out the `response` variable value and then the word `.sigAlg`.

Reply:
Good catch. Surrounded it in curlys.